lib/ensemble.dart

```diff
@@ -47,22 +47,12 @@ class Ensemble {
     externalScreenWidgets = widgets;
   }
 
-  final List<Map<String, Object?>> _afterInitMethods = [];
-  void addCallbackAfterInitialization(
-      {required Function method,
-      List<dynamic>? positionalArgs,
-      Map<String, dynamic>? namedArgs}) {
-    final function = {
-      'id': math.Random().nextInt(100000),
-      'method': method,
-      'positionalArgs': positionalArgs,
-      'namedArgs': namedArgs
-    };
-    _afterInitMethods.add(function);
+  final List<Function> _afterInitMethods = [];
+  void addCallbackAfterInitialization({required Function method}) {
+    _afterInitMethods.add(method);
   }
 
-  List<Map<String, Object?>> getCallbacksAfterInitialization() =>
-      _afterInitMethods;
+  List<Function> getCallbacksAfterInitialization() => _afterInitMethods;
```

Comment:
To prevent duplications, can you change this to a Set? if the same function is added twice my accident, we only call once.

Reply:
Done. Can you check again?